src/layouts/hass-tabs-subpage.ts

```diff
@@ -55,32 +55,34 @@ class HassTabsSubpage extends LitElement {
           (!page.advancedOnly || showAdvanced)
       );
 
-      return shownTabs.map(
-        (page, index) => html`
-          <div
-            class="tab ${classMap({
-              active: index === activeTab,
-            })}"
-            @click=${this._tabTapped}
-            .path=${page.path}
-          >
-            ${this.narrow
-              ? html`
-                  <ha-icon .icon=${page.icon}></ha-icon>
-                `
-              : ""}
-            ${!this.narrow || index === activeTab
-              ? html`
-                  <span class="name"
-                    >${page.translationKey
-                      ? this.hass.localize(page.translationKey)
-                      : name}</span
-                  >
-                `
-              : ""}
-            <mwc-ripple></mwc-ripple>
-          </div>
-        `
+      return tabs.map((page, index) =>
+        shownTabs.find((t) => page.path === t.path)
```

Comment:
This is expensive. I suggest we keep the old code but either make `activeTab` be the `path`, or compute `activePath` based on `activeTab` prior to returning this.

Reply:
:ok_hand: